command.go

```diff
@@ -202,6 +205,12 @@ type Command struct {
 	errWriter io.Writer
 }
 
+// Context returns underlying command context. If command wasn't
+// executed with ExecuteContext the returned context will be nil.
+func (c *Command) Context() context.Context {
+	return c.ctx
```

Comment:
Generally you do not want to have a nil context.
Instead I'd recommend we make sure that context is at least `context.Background()`.

Reply:
That's fair, I was thinking about that but didn't have strong opinions on that. But I changed it to use context.Background instead of nil. Thank you!